lib/libyui_client/widgets/table.rb

```diff
@@ -127,7 +147,7 @@ def selected_items
       private
 
       def get_index(column)
```

Comment:
I guess there is not need now for private function.

Reply:
This is not used anywhere else, so I would leave it for now. As display order of the columns might differ to the respective model, I believe there is no sense to provide this method to the client.

Reply:
I was referring to delete the function and just use it where is called as you already encapsulated the other method and index in core one.

Reply:
Let's leave it this way for now, as we might consider making it public, as for rows we have basically same issue, as row number might differ from display order, even if sorting was not changed.